deploy/helm/sumologic/values.yaml

```diff
@@ -311,7 +311,19 @@ fluent-bit:
       Systemd_Filter  _SYSTEMD_UNIT=var-lib-etcd2.service
       Max_Entries     1000
       Read_From_Tail  true
-
+    [INPUT]
+      Name              tail
+      Path              /var/log/kube-apiserver-audit.log
+      Parser            docker
+      Tag               audit.*
+      Refresh_Interval  5
+      Rotate_Wait       60
+      Mem_Buf_Limit     35MB
+      Buffer_Chunk_Size 2MB
+      Buffer_Max_Size   10MB
```

Comment:
@rvmiller89 @frankreno Do we want to allow this buffer limit? I would recommend keeping the same as for the container logs input section.

Reply:
+1 to keeping the same for consistency

Reply:
So are you suggesting that we set the " Mem_Buf_Limit    5MB" and then remove the "Buffer_Chunk_Size" and "Buffer_Max_Size" or where do you want them set to?

Reply:
Note that i got these setting from the upstream fluent bit helm chart, for what they are setting when tailing the kube-audit logs. See https://github.com/helm/charts/tree/master/stable/fluent-bit#configuration

Reply:
We should probably be consistent with upstream...